about.md

```diff
@@ -1,11 +1,9 @@
 ---
 layout: page
-title: About
+title: About Me
 permalink: /about/
 ---
 
-This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
+<p align="center"><img height="100px" width="100px" src="../assets/img/me_thumbnail.png"></p>
```

Comment:
There's a cleaner way to call assets within Jekyll without hard coding. Check the doc.